test.js

```diff
@@ -141,6 +141,40 @@ test('enqueue - add a message to the queue', t => {
   })
 })
 
+test('enqueue - stringify userId', t => {
+  const client = createClient()
+
+  client.track({
+    userId: 157963456373623802,
```

Comment:
At the time you pass `157963456373623802` number, it's already rounded down, so stringifying won't preserve the intended value: https://runkit.com/vdemedes/5a1d33234f113b00113d3b4f.

Reply:
Updated the test!